packages/models-library/src/models_library/services.py

```diff
@@ -482,6 +483,11 @@ class ServiceDockerData(ServiceKeyVersion, _BaseServiceCommonDataModel):
         alias="boot-options",
         description="Service defined boot options. These get injected in the service as env variables.",
     )
+    min_visible_inputs: NonNegativeInt | None = Field(
+        None,
+        alias="min-visible-inputs",
+        description="The number of 'data type inputs' displayed by default in the UI.",
```

Comment:
MINOR: define what means ``None`` ... ie. if set to ``None`` then all inputs are visible
TIP:  you can make it more readable with
```python

ALL = None

min_visible_inputs: NonNegativeInt | ALL = Field( ALL, .. )



```

Reply:
some inputs on these? :-)